daemon/daemon.go

```diff
@@ -506,14 +516,17 @@ func (daemon *Daemon) GetByName(name string) (*Container, error) {
 	return e, nil
 }
 
-func (daemon *Daemon) Children(name string) (map[string]*Container, error) {
+// Children returns all child containers of the given name. The
+// containers are returned as a map from the container name to a
+// pointer to Container.
```

Comment:
yes thanks. 

Now that I look at it again, I am not sure if the second sentence says anything more interesting than repeating the return signature.